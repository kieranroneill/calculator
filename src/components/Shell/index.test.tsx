import { shallow, ShallowWrapper } from 'enzyme';
import * as React from 'react';

// Components.
import { Shell, Props } from './';

interface Scope {
    props: Props;
    wrapper: ShallowWrapper;
}

describe('src/components/Shell', () => {
    let scope: Scope;

    beforeEach(() => {
        const props: Props = {
            children: <h1>Hello human</h1>,
        };

        scope = {
            props,
            wrapper: shallow(<Shell {...props} />),
        };
    });

    describe('<Shell /> snapshots', () => {
        it('should match the snapshot with the default props', () => {
            expect(scope.wrapper).toMatchSnapshot();
        });
    });
});
