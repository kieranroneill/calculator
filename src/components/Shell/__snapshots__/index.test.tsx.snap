// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/Shell <Shell /> snapshots should match the snapshot with the default props 1`] = `
<styled.div>
  <h1>
    Hello human
  </h1>
</styled.div>
`;
